packages/ebayui-core/src/components/ebay-video/component.ts

```diff
@@ -175,20 +233,33 @@ class Video extends Marko.Component<Input, State> {
         this.state.failed = true;
         this.state.isLoaded = true;
         this.playButtonContainer.remove();
-
+        
         this.emit("load-error", err);
     }
 
     showControls() {
         const copyConfig = Object.assign({}, videoConfig);
         copyConfig.controlPanelElements = [...videoConfig.controlPanelElements];
+        
+        if(this.input.layout === "compact") {
+            copyConfig.addBigPlayButton = false;
+            copyConfig.addSeekBar = false;
+            copyConfig.controlPanelElements = compactLayoutControlPanelElements;
+        }
```

Comment:
Can we instead have an object that we set copyConfig to that has all these? 
Like a global object of `compactConfig` 
And if input.layout === "compact" then we just do:
`copyConfig = Object.assign({}, videoCompactConfig)`